main.jsx

```diff
@@ -571,18 +573,24 @@ function Footer() {
   );
 }
 
+const ssrOptions = {
+  tw: {
+    theme: { colors },
```

Comment:
Are you able to overwrite `orange-500` to `#fc7c00`? I've tested it out and feel it fits decently with the other colors in the set.

Reply:
Done.